src/extension/background.js

```diff
@@ -230,6 +230,14 @@ const applyTheme = ({ theme }) => {
     newTheme.colors.tab_loading = colorToCSS(theme.colors.tab_line);
   }
 
+  if (!theme.colors.hasOwnProperty("popup")) {
```

Comment:
If users have previously built themes, we should take a guess at the right colors for popups and popup text